opensrp-giz-malawi/src/main/java/org/smartregister/giz/processor/GizMalawiProcessorForJava.java

```diff
@@ -160,6 +166,9 @@ public void processClient(List<EventClient> eventClients) throws Exception {
                     processService(eventClient, serviceTable);
                 } else if (eventType.equals(ChildJsonFormUtils.BCG_SCAR_EVENT)) {
                     processBCGScarEvent(eventClient);
+                } else if (eventType.equals(GizConstants.EventType.REASON_FOR_DEFAULTING)) {
+                    Timber.e("creating Reason for DAFAULTING Here");
```

Comment:
you can remove this now right or use `Timber.d`

Reply:
Removed